tests/backup_restore_test.go

```diff
@@ -424,39 +513,43 @@ func doBackup(ctx context.Context, driver dbplugin.Driver, database string) (*by
 }
 
 // Concurrently update a single row to mimic the ongoing business workload.
-// Returns the timestamp after inserting the initial value so we could check the PITR is done right.
-func startUpdateRow(ctx context.Context, t *testing.T, database string, port int) int64 {
+// Returns the timestamp after inserting the initial value so we could check the PITR is done right, passes the current id value into done chan when the update is finished.
+func startUpdateRow(ctx context.Context, t *testing.T, database string, port int, createAndInit bool, done chan<- int) int64 {
 	a := require.New(t)
 	db, err := connectTestMySQL(port, database)
 	a.NoError(err)
 
 	t.Log("Start updating data concurrently")
-	_, err = db.Exec("CREATE TABLE _update_row_ (id INT PRIMARY KEY);")
-	a.NoError(err)
+	if createAndInit {
```

Comment:
Can we use `CREATE IF NOT EXIST` and `REPLACE INTO` to remove the need for `createAndInit`?

Reply:
I think we can think about it. If we use the `CREATE IF NOT EXIST` statement, then the same statement is stored in the binlog. At this point, if `_update_row_` exists, it will not report an error, but when this table exists, I think we should report an error.

Reply:
How about we drop the table on stopping the update operation?
The goal is to make the test code simplier.

Reply:
There is a bug, when we do the first PITR, we restore a _update_row_ table. It will report `Error: TABLE _update_row_ already exists` when we try to test the second PITR if we `DROP TABLE _update_row_` in `select case <-ctx.Done()` case.  I prefer to add `createAndInit` as an argument. How about your opinion?

Reply:
Second thought: Can `CREATE TABLE _update_row_ be placed in `initPITRDB()`?